lib/customer_io.dart

```diff
@@ -88,9 +89,9 @@ class CustomerIO {
   /// @param userId unique identifier for a profile
   /// @param traits (Optional) params to set profile attributes
   void identify(
-      {required String userId,
-      Map<String, dynamic> traits = const {}}) {
-    return _platform.identify(userId: userId, traits: traits);
+      {required String userId, Map<String, dynamic> traits = const {}}) {
+    return _platform.identify(
+        userId: userId, traits: traits.excludeNullValues());
```

Comment:
Excluded `null` from all currently implemented functions, as we don't currently support `null` attributes in native SDKs. Open to any suggestions if there are other ideas on how to handle this!